### PR TITLE
Fix a bunch of misspellings

### PR DESCRIPTION
**Pull request by quackitsquinn**
Fixes a bunch of typos that have been dotted around the project

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Limelight.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public Limelight(String name) {
     limelightHasTarget = Shuffleboard.getTab("Driver").add(name + "has target", false).getEntry();
   }
 
-  /* now its time for getter method chaingun, which I have to write manually because VS Code */
+  /* now its time for getter method chain gun, which I have to write manually because VS Code */
```
Comment (Berdenson) [inline src/main/java/frc/robot/subsystems/Limelight.java:76]: you put a hyphen in the last one. Be consistent.

**Review by Berdenson — CHANGES_REQUESTED** [review]
do better

File: src/main/java/frc/robot/subsystems/Limelight.java (hunk 1 above)
Comment (quackitsquinn) [reply src/main/java/frc/robot/subsystems/Limelight.java]: ugh

**Review by Berdenson — APPROVED** [review]
speling is cool

**Pull request merged**